test/unit/document/document_test.ts

```diff
@@ -1187,11 +1188,27 @@ describe('Document', function () {
     );
   });
 
-  it('Get the value of the counter', function () {
+  it('gets the value of the counter', function () {
     const doc = Document.create<{ counter: Counter }>('test-doc');
     doc.update((root) => {
       root.counter = new Counter(155);
     });
     assert.equal(155, doc.getRoot().counter.getValue());
   });
+
+  it('stringifies any type of rich text attributes to JSON parsable string', function () {
+    const doc = Document.create<{ rich: RichText<Indexable> }>('test-doc');
+    doc.update((root) => {
+      root.rich = new RichText();
+      root.rich.edit(0, 0, 'aaa', { bold: true });
+      root.rich.setStyle(0, 3, { italic: true });
+      root.rich.setStyle(0, 3, { italic: null });
+      root.rich.setStyle(0, 3, { indent: 1 });
+      root.rich.setStyle(0, 3, { color: 'red' });
+    });
+    assert.equal(
+      '{"rich":[{"attrs":{""bold"":"true",""italic"":"null",""indent"":"1",""color"":"\\"red\\""},"content":"aaa"},{"attrs":{},"content":"\\n"}]}',
```

Comment:
This value doesn't seem to be valid JSON. 🤔

```suggestion
      '{"rich":[{"attrs":{"bold":"true","italic":null,"indent":1,"color":"red"},"content":"aaa"},{"attrs":{},"content":"\\n"}]}',
```

It seems that the value of the attribute is escaped twice when converting to JSON.